packages/x-tree-view/src/internals/plugins/useTreeViewFocus/useTreeViewFocus.types.ts

```diff
@@ -21,7 +21,7 @@ export interface UseTreeViewFocusParameters {
    * @param {string} nodeId The id of the node focused.
```

Comment:
I think "focused item" is more correct than "item focused" but feel free to ignore the word order change if you disagree

```suggestion
   * @param {string} nodeId The id of the focused item.
```